src/libs/tmdk/src/commands/init.ts

```diff
@@ -201,11 +194,10 @@ async function import_scenes_and_models(
                       Prefix: prefix,
                     })
                   );
-                  //console.log(objlist);
-                  if (objlist.hasOwnProperty("Contents")) {
-                    const contents = objlist["Contents"] as any[];
+                  if (objlist["Contents"] != undefined) {
+                    const contents = objlist["Contents"];
                     for (const [, value] of Object.entries(contents)) {
-                      if (value.hasOwnProperty("Key")) {
+                      if (value["Key"] != undefined) {
```

Comment:
What is the motivation for not using the native method? What if key is `never` or some other null-case value that is not `undefined`?

Reply:
This was to conform to an ESLint rule, perhaps I should revert and silence the rule for this line?

Reply:
After offline discussion seems the eslint rule, no-prototype-builtins, is related to inherited json properties. We can use `"Key" in value`. See second answer of below post:
https://stackoverflow.com/questions/20804163/check-if-a-key-exists-inside-a-json-object.